sections/dashboard/Overview/Overview.tsx

```diff
@@ -39,9 +48,116 @@ const Overview: FC = () => {
 	);
 	const [activeMarketsTab, setActiveMarketsTab] = useState<MarketsTab>(MarketsTab.FUTURES);
 
+	const { network, walletAddress, synthsMap } = Connector.useContainer();
+
+	const { tokenBalances } = useAppSelector(({ balances }) => balances);
+	// Only available on Optimism mainnet
+	const oneInchEnabled = network.id === 10;
+
+	const isL2 = useIsL2();
+
+	const [exchangeTokens, setExchangeTokens] = useState<any>([]);
+	const [kwentaBalance, setKwentaBalance] = useState(zeroBN);
+
+	useContractRead({
+		address: KWENTA_TOKEN_ADDRESS['10'],
+		abi: erc20ABI,
+		functionName: 'balanceOf',
+		args: [walletAddress!],
+		watch: false,
+		enabled: !!walletAddress && isL2,
+		onSettled(data, error) {
+			if (error) logError(error);
+			if (data) {
+				setKwentaBalance(weiFromWei(data));
+			}
+		},
+	});
+
+	const noKwentaFound = !Object.values(tokenBalances).find(
+		(token: any) => token.token.address.toLowerCase() === KWENTA_TOKEN_ADDRESS['10'].toLowerCase()
+	);
+
+	useEffect(() => {
+		(async () => {
+			const kwentaTokenObj = {
+				Kwenta: {
+					balance: kwentaBalance,
+					token: {
+						address: KWENTA_TOKEN_ADDRESS['10'].toLowerCase(),
```

Comment:
As we reference this a lot it would be nice to make it a const e.g. `const kwentaAddress = KWENTA_TOKEN_ADDRESS['10'].toLowerCase()`